ast/src/expression/value.rs

```diff
@@ -32,6 +33,7 @@ pub enum ValueExpression {
         #[serde(with = "crate::common::tendril_json")] StrTendril,
         Span,
     ),
+    String(Vec<char>, Span),
```

Comment:
Can we just use a rust `String` here? `Vec<char>` is 4x larger generally and much more cumbersome to use.